flow/block-inline/inline.html

```diff
@@ -63,15 +63,15 @@
 
   <body>
     <section>
-      <p>
+      <p class="editable">
```

Comment:
```suggestion
      <p>
```

Reply:
Without `editable` class name for `p` tag, the changes in text area are not reflected since it is used in the `fillCode` function.

Reply:
The `editable` class name is used only for CSS. In the HTML case, the `section` variable is used. Once you remove CSS, you don’t need `editable` anymore. It would be good to remove it from JS as well.